descriptors/ModuleDescriptor-template.json

```
@@ -104,6 +104,44 @@
         }
       ]
     },
+    {
+      "id": "circulation-settings-storage",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/circulation-settings-storage/circulation-settings",
+          "permissionsRequired": [
+            "circulation-settings-storage.circulation-settings.collection.get"
```

```suggestion
            "circulation-storage.circulation-settings.collection.get"
```
Please rename permissions in other endpoints below as well